sdks/python/apache_beam/runners/portability/portable_stager_test.py

```diff
@@ -64,7 +64,9 @@ def _stage_files(self, files):
     test_port = server.add_insecure_port('[::]:0')
     server.start()
     stager = portable_stager.PortableStager(
-        grpc.insecure_channel('localhost:%s' % test_port))
+        artifact_service_channel=grpc.insecure_channel(
+            'localhost:%s' % test_port),
+        staging_session_token='token')
```

Comment:
Ditto - verify that it's present in the requests

Reply:
done